### PR TITLE
[cosmetics] replace "movie" with "film"

### PR DESCRIPTION
**Pull request by MartijnKaijser**
Films is more common used in English (UK)

### CODE REVIEW

File: addons/resource.language.en_gb/resources/strings.po [hunk 1]
```diff
@@ -3844,7 +3844,7 @@ msgctxt "#1274"
 msgid "AirTunes"
 msgstr ""
 
-#. Filter movies/tvshows/music/episodes/artists/musicvideos/albums/songs
+#. Filter films/tvshows/music/episodes/artists/musicvideos/albums/songs
```
Comment (phil65) [inline addons/resource.language.en_gb/resources/strings.po]: I would keep movies for those since that is the content type "id" in core.

**Comment by Montellese** [comment]
Isn't this gonna cause a lot of inconsistencies with skin/addon specific strings that all currently use "movie" and "movies"?

**Comment by phil65** [comment]
@Montellese true, that would definitely be the case. In general everyone uses "Movies".

**Comment by MartijnKaijser** [comment]
Possibly. But if we are gonna take that in account we should also introduce all the spelling errors and wrong sentences.

**Comment by MartijnKaijser** [comment]
Same could be said about "shows/TV shows/series", "guide/tv guide/epg",

**Comment by phil65** [comment]
We also use movies in wiki / forums etc all over the place and all other software I know of also calls it "movies", too.

**Comment by vonH** [comment]
The wiki and other areas use a mixture of en-gb and en-us...

**Comment by Memphiz** [comment]
I can only add one thing to this. This is the same thing like calling "Serien" in german "Fernsehserien" - but do whatever you want with english. Film  <- sounds really unnatural to me. (doesn't matter whats common - XBMC/Kodi made the word "Movie" common for sure ;)).

**Comment by jjd-uk** [comment]
This change is wrong in my view, Movie is the more widely used term in modern English so it should stay as it is.

**Comment by uNiversaI** [comment]
Film is not entirely wrong, but its not as commonly used in the whole of the UK.
The Irish tend to use Film more than Brits do. In this general context where you can have home movies and such, film wouldn't even cover that scenario.

**Comment by jjd-uk** [comment]
What I meant is that the justification that Film is more common is wrong in my view.

**Comment by Montellese** [comment]
> Possibly. But if we are gonna take that in account we should also introduce all the spelling errors and wrong sentences back into core

No offence but that doesn't make any sense. Skin/Addon authors don't do spelling errors and wrong sentences on purpose (and neither do we but they are/were still there because not everyone is a native speaker). But they use "movie" and "movies" on purpose because that's what we use in core. So this comparison is heavily flawed.

**Comment by vonH** [comment]
uNiversal - I've not seen any what you say in 30 years.

**Comment by zag2me** [comment]
Just agreeing with closing this as Movie is actually more common.

**Pull request closed without merging**